src/wsi/win32/wsi_window_win32.cpp

```diff
@@ -211,11 +212,18 @@ namespace dxvk::wsi {
     }
 
     // Restore window position and apply the style
-    const RECT rect = pState->rect;
+    UINT swp_flags = SWP_FRAMECHANGED | SWP_NOACTIVATE;
+    RECT rect;
```

Comment:
Any reason rect can't just always be `pState->rect` with SWP_NOSIZE and SWP_NOMOVE?